### DISCUSSION TITLE
multiple glob patterns

### DISCUSSION THREAD

**Discussion by jdion84** [oven-sh/bun#8150] [Q&A]
is it possible to use multiple patterns with `Glob` somehow?

e.g.:

```ts
const commands = new Glob([
    './node_modules/my-package/commands/**/*.tsx',
    './app/commands/**/*.tsx',
])
```

**Comment by JulianCataldo** [comment]
I'd like too to achieve that too with `Bun.Glob(['stuff', '!foo'])`, for a drop-in replacement for :

- https://www.npmjs.com/package/fast-glob
- https://www.npmjs.com/package/glob
